api/scripts/createDB.sql

```diff
@@ -0,0 +1,13 @@
+DROP DATABASE IF EXISTS jobs_db;
+CREATE DATABASE jobs_db;
+
+\c jobs_db

```

Comment:
Not sure if this is just something I haven't seen before, but I would use 'use jobs_db;' here